cosmo_tester/test_suites/test_manager_status/manager_service_test.py

```diff
@@ -247,3 +247,26 @@ def get_node_count(url):
             'Verifying that both nodes are running but not clustered...')
         self.assertEqual(get_node_count(node1_url), 1)
         self.assertEqual(get_node_count(node2_url), 1)
+
+    def test_rabbitmq_policies(self):
+        """Tests that rabbitmq policies are set accordingly.
+        """
```

Comment:
the order of the policies matter.. and they should be tested according to the order in which they were created